pkg/testing/tools/tools.go

```diff
@@ -148,17 +148,35 @@ func InstallAgentForPolicyWithToken(ctx context.Context, t *testing.T,
 
 	// Don't check fleet status if --delay-enroll
 	if installOpts.DelayEnroll {
-		return nil
+		// agentID is not returned if delay enroll is used as it has not actually enrolled
```

Comment:
since the returned variables here can be omitted based on the installOpts, a function godoc seems appropriate to capture this

Reply:
I added a docstring for this.